lib/fakefs/file.rb

```diff
@@ -280,14 +280,14 @@ def self.executable?(filename)
     def self.chown(owner_int, group_int, filename)
       file = FileSystem.find(filename)
       if owner_int && owner_int != -1
-        owner_int.is_a?(Fixnum) || fail(TypeError,
-                                        "can't convert String into Integer")
-        file.uid = owner_int
+        owner_int.respond_to?(:to_i) || fail(TypeError,
```

Comment:
Updated. I also specified the base in Integer(), which is probably not important for any of the numbers that will realistically pass to this method but feel like it is probably the correct usage.
It was mentioned here, when I will looking up the usage of Integer:
https://moeffju.net/blog/integer-vs-to_i-in-ruby